SPECS/kubevirt/kubevirt.spec

```diff
@@ -18,28 +18,29 @@
 
 Summary:        Container native virtualization
 Name:           kubevirt
-Version:        0.59.0
-Release:        14%{?dist}
+Version:        1.2.0
+Release:        1%{?dist}
 License:        ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          System/Management
 URL:            https://github.com/kubevirt/kubevirt
 Source0:        https://github.com/kubevirt/kubevirt/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
-Source1:        disks-images-provider.yaml
```

Comment:
Where are these sources being used?

Reply:
those are used in the %install section

Reply:
actually, seems kubevirt_containers_meta is in older versions of kubevirt.spec but it was not included previously. so I guess it will be safe to remove them